samples/features/reporting-services/ssrs-migration-rss/ssrs_migration.rss

```diff
@@ -948,21 +948,27 @@ End Function
 
 'Helper function to construct correctly formatted filename
 Function GetSnkFilename(srcFilename As String, type As String) As String
-	Dim hasFE As Boolean = HasFileExtension(srcFilename)
-	If SnkIsNative And hasFE And Not GetExtension(type) = "" Then 'Cut off file extension, if present            
+	Dim currentFE As String = GetFileNameExtension(srcFilename)
+
+	If SnkIsNative And (GetExtension(type) = currentFE) and currentFE <> "" Then 'Cut off file extension, if present
```

Comment:
can you elaborate what is the problem and what is the proposed fix

for example when I try the old code in the IsNative code path with the example "One.Two.rdl" the srcFileName is One.Two which is what is expected regarding the comments.

Is the problem the logic of the removal of the characters after the final dot or is that GetExtension is case sensitive?

Reply:
Remove me